equinox/_module.py

```diff
@@ -324,7 +322,7 @@ def __new__(
                         converter_annotation = parameters[0].annotation
                         if converter_annotation is inspect.Signature.empty:
                             converter_annotation = Any
-                cls.__init__.__annotations__[f.name] = converter_annotation
+                    cls.__init__.__annotations__[f.name] = converter_annotation
```

Comment:
I think this is an unintentional indent?

Reply:
unintentional!

Reply:
Fixed